### PR TITLE
add support for sending emails

### PR DESCRIPTION
**Pull request by michaljirman**
add support for sending emails; tiny

### CODE REVIEW

**Comment by manicminer** [comment]
Thanks @michaljirman, this is looking good! Could you add a test for each of those client methods?

**Comment by michaljirman** [comment]
@manicminer I was thinking about it but it gets quite tricky to test. You need to use a special user with an ms365 licence for this. I am not sure how to create such a test user.

File: msgraph/me.go [hunk 1]
```diff
@@ -69,3 +69,24 @@ func (c *MeClient) GetProfile(ctx context.Context) (*Me, int, error) {
 	}
 	return &me, status, nil
 }
+
+// SendMail sends message specified in the request body.
```
Comment (manicminer) [inline msgraph/me.go:73]:
```suggestion
// SendMail sends message specified in the request body.
// TODO: Needs testing with an O365 user principal
```

File: msgraph/users.go [hunk 2]
```diff
@@ -172,3 +172,24 @@ func (c *UsersClient) ListGroupMemberships(ctx context.Context, id string, filte
 	}
 	return &data.Groups, status, nil
 }
+
+// SendMail sends message specified in the request body.
```
Comment (manicminer) [inline msgraph/users.go:176]:
```suggestion
// SendMail sends message specified in the request body.
// TODO: Needs testing with an O365 user principal
```

**Review by manicminer — APPROVED** [review]
@michaljirman That makes sense. Let's merge it without a test for now, and we can look at supporting tests with user principals (most likely using Azure CLI as mechanism) as there are going to be more resources with this requirement.

**Pull request merged**